hyperparameter_tuning/rapids_bring_your_own/rapids_sagemaker_hpo.ipynb

```diff
@@ -0,0 +1,1819 @@
+   "outputs": [],
+   "source": [
+    "estimator_params = {\n",
+    "    'image_name' : ecr_fullname,\n",
+    "    \n",
+    "    'train_use_spot_instances': use_spot_instances_flag,\n",
+    "    'train_instance_type' : instance_type, \n",
+    "    'train_instance_count' : 1,     \n",
+    "    \n",
+    "    'train_max_run'  : max_duration_of_experiment_seconds, # 24 hours \n",
+    "    'train_max_wait' : max_duration_of_experiment_seconds + 1,  \n",
+    "    \n",
+    "    'input_mode'  : 'File',\n",
+    "    'output_path' : s3_model_output,\n",
+    "    \n",
+    "    'sagemaker_session' : session,     \n",
+    "    'role' : execution_role,    \n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 36,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "WARNING:root:Parameter image_name will be renamed to image_uri in SageMaker Python SDK v2.\n"
+     ]
+    }
+   ],
+   "source": [
+    "estimator = sagemaker.estimator.Estimator( **estimator_params  )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<span style=\"color:#8735fb; font-size:20pt\"> 2.3 - Test Estimator </span>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we are ready to test by asking SageMaker to run the BYOContainer logic inside our Estimator. This is a useful step if you've made changes to your custom logic and are interested in making sure everything works before launching a large HPO search. \n",
+    "\n",
+    "> Note: This verification step will use the default hyper-parameter values declared in our custom train code, as SageMaker HPO will not be orchestrating a search for this single run."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 37,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "s3 data input    =\ts3://sagemaker-rapids-hpo-us-east-1/1_year\n",
+      "s3 model output  =\ts3://sagemaker-rapids-hpo-us-east-1/trained-models\n",
+      "compute          =\tsingleGPU\n",
+      "algorithm        =\tXGBoost, 3 cv-fold\n",
+      "instance         =\tml.p3.2xlarge\n",
+      "spot instances   =\tTrue\n",
+      "hpo strategy     =\tRandom\n",
+      "max_experiments  =\t8\n",
+      "max_parallel     =\t2\n",
+      "max runtime      =\t86400 sec\n"
+     ]
+    }
+   ],
+   "source": [
+    "summarize_choices( s3_data_input, s3_model_output, code_choice, algorithm_choice, \n",
+    "                   cv_folds, instance_type, use_spot_instances_flag, search_strategy, \n",
+    "                   max_jobs, max_parallel_jobs, max_duration_of_experiment_seconds )"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 38,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdin",
+     "output_type": "stream",
+     "text": [
+      "confirm test run? [ y / n ] :  y\n"
```

Comment:
It breaks after this:

```
ClientError: An error occurred (ValidationException) when calling the CreateTrainingJob operation: No S3 objects found under S3 URL "s3://sagemaker-rapids-hpo-us-east-1/1_year" given in input data source. Please ensure that the bucket exists in the selected region (us-west-2), that objects exist under that S3 prefix, and that the role "arn:aws:iam::082907515444:role/team-markham-a2i" has "s3:ListBucket" permissions on bucket "sagemaker-rapids-hpo-us-east-1". Error message from S3: The bucket is in this region: us-east-1. Please use this region to retry the request
```

Reply:
Thanks Aaron, looking into how to make the S3 bucket (currently in east-1) accessible across multiple regions. Hope to have a follow up shortly.

Reply:
- Replicated the S3 dataset bucket to us-west-2
- Added a region validation check 
- Added documentation to inform users that they need to run in either the us-west-2 or us-east-1 region